Mathlib/LinearAlgebra/RootSystem/Defs.lean

```diff
@@ -419,12 +419,20 @@ lemma root_eq_neg_iff :
   refine ⟨fun h ↦ P.root.injective ?_, fun h ↦ by simp [h]⟩
   rw [root_reflection_perm, reflection_apply_self, h, neg_neg]
 
+lemma root_eq_neg_iff' :
```

Comment:
Note to self: shouldn't this just replace the existing root_eq_nef_iff? (will investigate tomorrow)